widget/embedded/src/containers/QuoteInfo/QuoteInfo.styles.ts

```diff
@@ -14,3 +14,13 @@ export const FooterAlert = styled('div', {
   width: '100%',
   display: 'flex',
 });
+
+export const SkeletonContainer = styled('div', {
+  variants: {
+    paddingTop: {
+      true: {
+        paddingTop: '$10',
+      },
+    },
+  },
+});
```

Comment:
If this variable isn't being used anywhere, just go ahead and remove it.

Reply:
fixed